distro/run/modules/swaggerui/ui/webpack.config.js

```diff
@@ -27,10 +27,12 @@ const config = {
             patterns: [{from: 'src/index.html'}, {from: 'src/favicon.ico'}],
         }),
         new LicenseCheckerWebpackPlugin({
+            // https://github.com/microsoft/license-checker-webpack-plugin/issues/25
+            filter: /(^.*[/\\]node_modules[/\\]((?:@[^/\\]+[/\\])?(?:[^@/\\][^/\\]*)))/,
```

Comment:
```suggestion
            filter: /(^.*[/\\]node_modules[/\\]((?:@[^/\\]+[/\\])?[^@\/\\][^\/\\]*))/,
```

🙃 Unnecessary non-capturing group